style.css

```diff
@@ -0,0 +1,142 @@
+:root
+{
+    --dark-cyan: #0E8784;
+    --dark-grey-blue: #333D4B;
+    --pale-orange: #FDD6BA; 
+    --light-cream: #FEFCF7;
+    --grey: #83888F;
+    --white: #FFFFFF;
+
+    --fraunces: 'Fraunces', serif;
+    --barlow: 'Barlow', serif;
+}
+
+*
+{
+    box-sizing: border-box;
+}
+
+.page-container
+{
+    max-width: 1280px;
+    padding: 2rem;
+    margin: 0 auto;
+}
+
+body{
+    margin: 0;
+    padding: 0;
+    background-color: var(--light-cream);
+    font-family: var(--fraunces);
+    color: #333D4B;
+}
+
+button
+{
+    cursor: pointer;
+}
+
+/*COLLECTION*/
+
+.collection-container
+{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+.collection-container:not(:first-child)
+{
+    gap: 2.5rem;
+}
+
+.collection-container__img
+{
+    width: 15rem;
+}
+
+.collection-container__item
+{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+}
+
+.collection-container__content
+{
+    font-family: var(--barlow);
+    color: var(--dark-grey-blue);
+}
+
+.collection-container__title
+{
+    color: var(--dark-grey-blue);
+    text-transform: capitalize;
+}
+
+.collection-container__opacity-title
+{
+    opacity: 25%;
+    font-size: 2.5rem;
+    margin-bottom: 1rem;
+}
+
+@media(min-width:768px)
+{
+   .collection-container__item
+   {
+    flex-direction: row;
+    text-align: start;
+   }
+
+   .collection-container__title
+   {
+    margin-bottom: 2.5rem;
+   }
+
+   .collection-container__description-container
+   {
+    display: flex;
+    flex-direction: column;
+    max-width: 20rem;
+   }
+
+   .collection-container
+   {
+    position: relative;
+    gap: 3rem;
+   }
+
+   .collection-container__opacity-title
+   {
+    position: absolute;
+    top: -4.5rem;
+    font-size: 7.4rem;
+    z-index: -1;
+    opacity: 15%;
+   }
+}
+
+@media(min-width: 1280px)
+{
+    .collection-container__item
+   {
+    flex-direction: column;
+    text-align: center;
+   }
+   .collection-container
+   {
+    flex-direction: row;
+   }
+   .collection-container__title
+   {
+    margin: 4rem 0 1rem 0
```

Comment:
You can remove the last 0, because if margin/padding will have 3 values then:
- first is `margin-top`
- second is `margin-left` and `margin-right`
- third is `margin-bottom`